crates/sui/src/client_commands.rs

```diff
@@ -1401,6 +1401,40 @@ impl Display for SuiClientCommandResult {
                 table.with(style);
                 write!(f, "{}", table)?
             }
+            SuiClientCommandResult::Gas(gases) => {
+                if gases.is_empty() {
+                    write!(f, "No gas coins are owned by this address")?;
+                    return Ok(());
+                }
+
+                let mut builder = TableBuilder::default();
+                builder.set_header(vec!["gasCoinId", "coinValue"]);
+                for coin in gases {
+                    builder.push_record(vec![
+                        coin.gas_coin_id.to_string(),
+                        coin.coin_value.to_string(),
+                    ]);
+                }
+                let mut table = builder.build();
+                table.with(TableStyle::rounded());
+                if gases.len() > 10 {
+                    table.with(TablePanel::header(format!(
+                        "Showing {} gas coins and their values.",
+                        gases.len()
+                    )));
+                    table.with(TablePanel::footer(format!(
```

Comment:
do we need this info in the header and footer? I think just the header would be sufficient

Reply:
Great question. While not very often, there are addresses that own lots of objects/coins. And at times, you get a long list (100+), so when you issue the command you cannot see them all at once, and you are only shown the last X of them. Having a footer gives you an idea of how many are there and allows the user to have a quick idea of what are they dealing with. I personally like having both a footer and a header for long tables, but if you think it's not needed, we can definitely remove it.